### PR TITLE
Change suppliers select into bootstrap dropdown

### LINKED ISSUE #9669: [BOOM-5318] Supplier list module
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5318).

- _**Reporter:**_ marion_francois
- _**Created at:**_ Fri, 13 Apr 2018 17:10:59 +0200

<p>In supplier list module, I can choose to display suppliers in plain text or drop-down.</p>

<p>If I choose drop-down, it's not very pretty</p>
        
capture du 2018-04-13 17-07-04

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The select of supplier module on category page is ugly, let's use a bootstrap dropdown we already use on this page
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9669 
| How to test?  | Install the ps_supplierlist module, then change the configuration to dropdown instead of plain text

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16526)

### CODE REVIEW

File: themes/classic/modules/ps_supplierlist/views/templates/_partials/supplier_form.tpl [hunk 1]
```diff
@@ -23,11 +23,25 @@
...
-    <option value="">{l s='All suppliers' d='Shop.Theme.Catalog'}</option>
+<div class="suppliers-sort dropdown">
+  <button
+    class="btn-unstyle select-title"
+    rel="nofollow"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="false">
+    {l s='All suppliers' d='Shop.Theme.Catalog'}
+    <i class="material-icons float-xs-right">&#xE5C5;</i>
```
Comment (PierreRambaud) [inline themes/classic/modules/ps_supplierlist/views/templates/_partials/supplier_form.tpl]:
```suggestion
    <i class="material-icons float-xs-right">arrow_drop_down</i>
```

**Review by PierreRambaud — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is OK.
image

PS: the ps_supplierlist is not working.

Thanks!

**Comment by Progi1984** [comment]
@NeOMakinG Hi, could you rebase your PR before we merge it ? Thanks

**Review by Progi1984 — APPROVED** [review]
Approved initially by @PierreRambaud

**Comment by PierreRambaud** [comment]
Thanks @NeOMakinG

**Pull request merged**